modules/nw-sriov-configuring-operator.adoc

```diff
@@ -128,6 +144,22 @@ $ oc patch sriovoperatorconfig default --type=merge \
   -n openshift-sriov-network-operator \
   --patch '{ "spec": { "enableOperatorWebhook": <value> } }'
 ----
++
+[TIP]
```

Comment:
do we need to provide another this way to update the parameter?
I think above `oc patch` is enough and it's easy to update it.

Reply:
anyway,  it's also ok for me

Reply:
@zhaozhanqi , while the `oc patch` is perfectly clear and easy, the goal for the PR is to show customers that they can use YAML--maybe in a Git repo--to manage the cluster declaratively.